features/admin.users.v1/components/user-profile.tsx

```diff
@@ -161,7 +167,7 @@ interface UserProfilePropsInterface extends TestableComponentInterface {
      */
     editUserDisclaimerMessage?: ReactNode;
     /**
-     * Admin user type
+     * Admin user typeisLoading: isAuthenticatedUserFetchRequestLoading
```

Comment:
isloading property is not properly added it seems

Reply:
82d4ff507d55ec7ed6c0b57080b6e89963594857